next/lib/utils.ts

```diff
@@ -56,3 +56,25 @@ export const getFileType = (file: string) => {
   }
   return null;
 };
+
+export const makePathAbsolute = (path: string) =>
+  // eslint-disable-next-line n/no-process-env
+  process.env.NEXT_PUBLIC_FRONTEND_URL + path;
```

Comment:
The `eslint-disable-next-line` can be removed if we use our typed env setup:

```tsx
import { env } from "@/env";

const makePathAbsolute = (path: string) => env.NEXT_PUBLIC_FRONTEND_URL + path;
```

Reply:
for some reason, this didn't work for me... can you try and see it if works for you?

Reply:
Sure, I pushed [`04a799d` (#209)](https://github.com/wunderio/next-drupal-starterkit/pull/209/commits/04a799d22182dec761e59f57c62d4795cd902319) and it seems to work